### PR TITLE
Bic-6057 update auth only dev doc visa requirements

### PR DESCRIPTION
**Pull request by jwebster-worldline**

### CODE REVIEW

**Review by Darren-Worldline — APPROVED** [review]

File: source/docs/guides/3D_secure_2_0.md [hunk 1]
```diff
@@ -612,6 +614,9 @@ All the browser values can be collected via JavaScript, unless of course they ha
 |card.number|String|The credit card number to authenticate against|
 |card.expiry.month|Number|The two digit month of the card expiration date.|
 |card.expiry.year|Number|The four digit year of the card expiration date.|
+|card.name|String|Name of the cardholder *Mandatory for Visa|
```
Comment (gkarmadiWorldline) [inline source/docs/guides/3D_secure_2_0.md:617]: should we say: Name of the cardholder is *mandatory for Visa

**Review by gkarmadiWorldline — CHANGES_REQUESTED** [review]
Hi Jack, please review my comment

File: source/docs/guides/3D_secure_2_0.md (hunk 1 above)
Comment (jwebster-worldline) [reply source/docs/guides/3D_secure_2_0.md:617]: No I don't think so, "Name of the cardholder" and "Mandatory for Visa" are 2 different statements, the second is an additional note on the end of the description

**Review by gkarmadiWorldline — APPROVED** [review]
approved

**Pull request merged**